src/Camelotia.Presentation.Avalonia/Camelotia.Presentation.Avalonia.csproj

```diff
@@ -32,4 +32,20 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Views\FolderView.xaml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Views\FolderView.xaml.cs">
+      <DependentUpon>FolderView.xaml</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Views\FolderView.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
I guess these lines were added accidentally by an IDE. We have a glob pattern currently https://github.com/worldbeater/Camelotia/blob/main/src/Camelotia.Presentation.Avalonia/Camelotia.Presentation.Avalonia.csproj#L26 so including the files explicitly isn't necessary.

Reply:
Good catch - just Visual Studio being Visual Studio :)